docs/code-search/code-navigation/precise_code_navigation.mdx

```diff
@@ -0,0 +1,81 @@
+
+Precise code navigation relies on the following technologies:
+
+- [SCIP](https://github.com/sourcegraph/scip) (SCIP Code Intelligence Protocol) and
+- [LSIF](https://github.com/Microsoft/language-server-protocol/blob/master/indexFormat/specification)
+(Language Server Index Format) data to deliver precomputed code navigation. It provides fast and highly accurate code navigation but needs to be periodically generated and uploaded to your Sourcegraph instance. Precise code navigation is an opt-in feature: repositories for which you have not uploaded indexes will continue to use the search-based code navigation.
+
+## Setting up code navigation for your codebase
+
+There are several options for setting up precise code navigation:
+
+1. **Manual indexing**. Index a repository and upload it to your Sourcegraph instance:
+
+    - [Index a Go repository](/code_navigation/how-to/index_a_go_repository#manual-indexing)
+    - [Index a TypeScript or JavaScript repository](/code_navigation/how-to/index_a_typescript_and_javascript_repository#manual-indexing)
+    - [Index a Java, Scala, or Kotlin repository](https://sourcegraph.github.io/scip-java/docs/getting-started.html)
+    - [Index a Python repository](https://sourcegraph.com/github.com/sourcegraph/scip-python)
+    - [Index a Ruby repository](https://sourcegraph.com/github.com/sourcegraph/scip-ruby)
+
+2. [**Automate indexing via CI**](/code_navigation/how-to/adding_lsif_to_workflows): Add indexing and uploading to your CI setup.
+3. [**Auto-indexing**](/code_navigation/how-to/enable_auto_indexing): Sourcegraph will automatically index your repositories and enable precise code navigation for them.
+4. Set up **auto-dependency indexing** to navigate and search through the dependencies your code uses:
+    - **Go**: Enable [auto-indexing](/code_navigation/explanations/auto_indexing) and Sourcegraph will start indexing your dependencies.
+    - **JavaScript, TypeScript**: Enable [auto-indexing](/code_navigation/explanations/auto_indexing) and set up an [npm dependencies code host](/integration/npm).
+    - **Java, Scala, Kotlin**: Enable [auto-indexing](/code_navigation/explanations/auto_indexing) and set up a [JVM dependencies code host](/integration/jvm).
+
+## Supported languages and indexers
+
+Precise Code Navigation requires language-specific indexes to be generated and uploaded to your Sourcegraph instance. We currently have precise code navigation support for the languages below. See the [indexers page](/code_navigation/references/indexers) for a detailed breakdown of each indexer's status.
+
+|        Language        |                                      Indexer                                      |        Status         |
+| ---------------------- | --------------------------------------------------------------------------------- | --------------------- |
+| Go                     | [lsif-go](https://sourcegraph.com/github.com/sourcegraph/lsif-go)                 | 🟢 Generally available |
```

Comment:
This should be scip-go, I think I updated that in main but since the file is moved/new it didnt map